### ISSUE TITLE
False positive - strings with comments

### ISSUE THREAD

**Issue by jamiehunt** [dral3x/StringsLint#20]
Strings with same-line comments appear to produce false missing violations
`Partial Violation: Localized string "my_string" is missing in locale "ja" (partial)`

**en/Localizable.strings**
```
"my_string" = "Foo bar...";
```

**ja/Localizable.strings**
```
"my_string" = "フーバー"; // Some comment
```

**Comment by dral3x** [comment]
Hi @jamiehunt !
Thanks for reporting this issue.

Are comments supposed to be added inline? I never saw them used like that inside a `.strings.` file.
